shim_test.go

```diff
@@ -147,3 +152,31 @@ func TestNewShimConfigFromUnknownShimPodConfig(t *testing.T) {
 
 	testNewShimConfigFromPodConfig(t, podConfig, nil)
 }
+
+func TestStopShimSuccessfulProcessNotRunning(t *testing.T) {
+	cmd := exec.Command(testRunningProcess, "0.001")
+	if err := cmd.Start(); err != nil {
+		t.Fatal(err)
+	}
+
+	pid := cmd.Process.Pid
+
+	if err := cmd.Wait(); err != nil {
+		t.Fatal(err)
+	}
+
+	if err := stopShim(pid); err != nil {
+		t.Fatal(err)
+	}
+}
+
+func TestStopShimSuccessfulProcessRunning(t *testing.T) {
+	cmd := exec.Command(testRunningProcess, "1")
```

Comment:
Maybe make it `999` to minimise any change of it finishing before the `stopShim()` call.